### PR TITLE
Adding Now Playing plugin

### PR DESCRIPTION
**Pull request by adampk90**

### CODE REVIEW

**Comment by keithamus** [comment]
Hey @adampk90. I've run this through [shellcheck](http://www.shellcheck.net/) and there are a few errors. Could you please address these?

File: Music/nowplaying.5s.sh [hunk 1]
```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# <bitbar.title>Now playing</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Adam Kenyon</bitbar.author>
+# <bitbar.author.github>adampk90</bitbar.author.github>
+# <bitbar.desc>Shows and controls the music that is now playing. Currently supports Spotify, iTunes, and Vox.</bitbar.desc>
+# <bitbar.image></bitbar.image>
```
Comment (keithamus) [inline Music/nowplaying.5s.sh]: Could you please add a screenshot also? Thanks :smile:

**Comment by adampk90** [comment]
Ah oops. I added an image to my working directory and it didn't make it into the push I guess. Also, thanks for pointing me toward shellcheck, that's neat.

**Comment by keithamus** [comment]
Cool :+1:. Funny coincidence: I added that Underoath album to my playlist just the other day :smile: :metal:

**Pull request merged**